kolibri/core/content/api.py

```diff
@@ -1382,7 +1396,9 @@ class UserContentNodeViewset(BaseContentNodeMixin, BaseValuesViewset, ListModelM
     A content node viewset for filtering on user specific fields.
     """
 
-    filter_backends = (DjangoFilterBackend,)
+    filter_backends = (DjangoFilterBackend, filters.OrderingFilter)
+    ordering_fields = ["last_interacted"]
+    ordering = ("lft", "id")
     filter_class = UserContentNodeFilter
     pagination_class = OptionalPagination
 
```

Comment:
Probably simplest to add a `get_queryset` method to this class and add the annotation to the queryset this method returns.